app/client/scss/components/_event-card.scss

```diff
@@ -5,6 +5,12 @@
   overflow: hidden;
 }
 
+.event-card2 {
```

Comment:
Be clearer with your class names. After looking through the file changes, I still have no idea what event-card2 is for. How is it different from event-card?